recipes/r-btyd/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = '2.4.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-btyd
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BTYD_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BTYD/BTYD_{{ version }}.tar.gz
+  sha256: 4be02c52b76f71556a3611cebca2378c9e8aec8f1f03a42c5931eb9facfcadae
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-matrix
+    - r-dplyr
+    - r-hypergeo
+    - r-optimx
+  run:
+    - r-base
+    - r-matrix
+    - r-dplyr
+    - r-hypergeo
+    - r-optimx
+
+test:
+  commands:
+    - $R -e "library('BTYD')"           # [not win]
+    - "\"%R%\" -e \"library('BTYD')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=BTYD
+  license: GPL-3
```

```suggestion
  license: GPL-3.0-only
```

cran meta data `# License: GPL-3` maps to https://spdx.org/licenses/GPL-3.0-only.html